nixos/modules/services/web-apps/funkwhale/funkwhale.nix

```diff
@@ -0,0 +1,329 @@
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  pythonEnv = (
+    let pythonRequirements = import ./requirements.nix { inherit pkgs; };
+    in
+      with pkgs; 
+      python36.withPackages(ps: with ps; (builtins.attrValues pythonRequirements.packages))
+    );
+  cfg              = config.services.funkwhale;
+  funkwhaleHome    = config.users.extraUsers.funkwhale.home;
+  funkwhaleEnvFile = (import ./funkwhale.env.nix)       { pkgs = pkgs; cfg = cfg; };
+  funkwhaleEnv = {
+    FUNKWHALE_ENV_FILE = "${funkwhaleEnvFile}";
+  };
+in 
+{ 
+
+  options = {
+    services.funkwhale = {
+      enable = mkEnableOption "funkwhale";
+
+      api_ip = mkOption {
+        type = types.str;
+        default = "127.0.0.1";
+        description = ''
+          Funkwhale API IP
```

Comment:
All description fields should end with a `.`

Reply:
fixed